src/components/compress/style.scss

```diff
@@ -3,20 +3,72 @@
   height: 100%;
   contain: strict;
   display: grid;
+  align-items: end;
+  align-content: end;
+  grid-template-rows: 1fr auto;
 
-  &.horizontal {
+  @media (min-width: 600px) {
     grid-template-columns: 1fr auto;
-    grid-template-rows: calc(100% - 75px);
+    grid-template-rows: 100%;
+  }
+}
+
+.options {
+  color: #fff;
+  opacity: 0.9;
+  font-size: 1.2rem;
+  display: flex;
+  flex-flow: column;
+  max-width: 400px;
+  margin: 0 auto;
+  width: calc(100% - 60px);
+  max-height: calc(100% - 143px);
+  overflow: hidden;
+
+  @media (min-width: 600px) {
+    max-height: calc(100% - 75px);
+    width: 300px;
+    margin: 0;
+  }
 
-    @media (min-width: 860px) {
-      grid-template-rows: 100%;
-    }
-    align-items: end;
-    align-content: end;
+  @media (min-width: 860px) {
+    max-height: none;
   }
+}
+
+.multi-panel {
+  position: relative;
+  display: flex;
+  flex-flow: column;
 
-  &.vertical {
-    // TODO: make the mobile view work
-    background: red;
+  // Reorder so headings appear after content:
```

Comment:
The multi-panel expects header-content-header-content, but design-wise we want the header at the bottom.